### ISSUE TITLE
Add status screen

### ISSUE THREAD

**Issue by s-macke** [s-macke/jor1k#24]
One of the main applications of jor1k is teaching. It would be therefore nice to add a status-screen with statistics about syscalls, page faults, exceptions, network usage, filesystem usage, and further debug messages.

**Comment by regular** [comment]
can we use dtrace for this?

**Comment by s-macke** [comment]
No, I wouldn't do it on Linux emulation level, but on the Javascript level.
A stats-object which can be accessed by every device. (like message). And the user can choose which details he wants to see. Especially interesting is the filesystem. Which files are downloaded and which are opened. A button next to the framebuffer to switch between the framebuffer and different debug outputs. At the moment the website is overloaded to do all this.There is so much more I would like to change on the current user interface.

**Comment by neelabhg** [comment]
This is awesome. I would love to help, let me know what I can do.

**Comment by s-macke** [comment]
Just start. You can't do much wrong, because I have no idea yet, how to implement it in a nice way.

**Comment by s-macke** [comment]
For the moment, it doesn't matter how the stats are displayed. If instead of the framebuffer, in the terminal or outside of terminal and framebuffer.

**Comment by benjamincburns** [comment]
Could see using a tty for this, or it'd be pretty awesome to send statsd messages from inside the image sent to an emulated daemon running alongside the main VM.

The latter would be a fair amount of effort but having the ability to write client side 'servers' in JS which run outside the emulator and interact via the network would probably open up all sorts of interesting things.

**Comment by benjamincburns** [comment]
Otherwise, I'm not sure how you can get stats on those things without _something_ living inside the image to give them to you.
